### ISSUE TITLE
Material-UI: The `css` function is deprecated. Use the `styleFunctionSx` instead.

### ISSUE THREAD

**Issue by davidhenley** [marmelab/react-admin#6570]
Forgive me if this is incorrect, but this is the warning I'm getting when using 3.18.1

Is this a problem with ra-material-ui using a deprecated prop?

```
  "dependencies": {
    "prop-types": "^15.7.2",
    "react": "^17.0.2",
    "react-admin": "^3.18.1",
    "react-dom": "^17.0.2",
    "react-scripts": "4.0.3"
  },
```

**Comment by fzaninotto** [comment]
Which react-admin component raises this warning? We're not using the `css` function in our codebase.

**Comment by davidhenley** [comment]
| css | @ | vendors~main.chunk.js:42951
-- | -- | -- | --
  | ./node_modules/@material-ui/core/esm/Box/Box.js | @ | vendors~main.chunk.js:2773
  | __webpack_require__ | @ | bundle.js:852
  | fn | @ | bundle.js:151
  | ./node_modules/@material-ui/core/esm/Box/index.js | @ | vendors~main.chunk.js:2794
  | __webpack_require__ | @ | bundle.js:852
  | fn | @ | bundle.js:151
  | ./node_modules/@material-ui/core/esm/index.js | @ | vendors~main.chunk.js:33538
  | __webpack_require__ | @ | bundle.js:852
  | fn | @ | bundle.js:151
  | ./node_modules/ra-ui-materialui/esm/auth/Login.js | @ | vendors~main.chunk.js:107765
  | __webpack_require__ | @ | bundle.js:852
  | fn | @ | bundle.js:151
  | ./node_modules/ra-ui-materialui/esm/auth/index.js | @ | vendors~main.chunk.js:108233
  | __webpack_require__ | @ | bundle.js:852
  | fn | @ | bundle.js:151
  | ./node_modules/ra-ui-materialui/esm/index.js | @ | vendors~main.chunk.js:118546
  | __webpack_require__ | @ | bundle.js:852
  | fn | @ | bundle.js:151
  | ./node_modules/react-admin/esm/AdminUI.js | @ | vendors~main.chunk.js:134304
  | __webpack_require__ | @ | bundle.js:852
  | fn | @ | bundle.js:151
  | ./node_modules/react-admin/esm/Admin.js | @ | vendors~main.chunk.js:134056
  | __webpack_require__ | @ | bundle.js:852
  | fn | @ | bundle.js:151
  | ./node_modules/react-admin/esm/index.js | @ | vendors~main.chunk.js:134368
  | __webpack_require__ | @ | bundle.js:852
  | fn | @ | bundle.js:151
  | (anonymous) | @ | main.chunk.js:34
  | ./src/App.js | @ | main.chunk.js:355
  | __webpack_require__ | @ | bundle.js:852
  | fn | @ | bundle.js:151
  | (anonymous) | @ | main.chunk.js:3948
  | ./src/index.js | @ | main.chunk.js:4045
  | __webpack_require__ | @ | bundle.js:852
  | fn | @ | bundle.js:151
  | 1 | @ | main.chunk.js:4059
  | __webpack_require__ | @ | bundle.js:852
  | checkDeferredModules | @ | bundle.js:46
  | webpackJsonpCallback | @ | bundle.js:33
  | (anonymous) | @ | main.chunk.js:1

**Comment by davidhenley** [comment]
Closed in favor of this issue: #6580
